docs/tutorial/index.md

```diff
@@ -1,74 +1,48 @@
-# Get Started with UP4W
+# Get started with UP4W
 
-In this tutorial you will learn all the basic things that you need to know to start taking advantage of your Ubuntu Pro subscription on Ubuntu WSL at scale with Ubuntu Pro for WSL (UP4W).
+Windows Subsystem for Linux ([WSL](https://ubuntu.com/desktop/wsl)) makes it possible to run Ubuntu — the number 1 open-source operating system — on a Windows machine.
+With Ubuntu Pro for WSL (UP4W) an [Ubuntu Pro](https://ubuntu.com/pro) subscription empowers you to manage Ubuntu WSL instances at scale.
 
-<br/>
-
-**What you'll need:**
-
-- A Windows 11 machine with access to the Microsoft Store and a minimum of 16GB of RAM and 8-core processor.
-
-- Familiarity with the Windows PowerShell.
-
-<br/>
-
-**What you'll do:**
-
-- Set things up
-
-- Watch UP4W transform your Ubuntu Pro game on WSL!
-
-- Tear things down
-
-<br/>
+In this tutorial you will develop an understanding of how UP4W can be installed and deployed for managing multiple WSL instances.
+To complete this tutorial you will need a Windows 11 machine with a minimum of 16GB RAM and a 8-core processor.
 
 ```{note}
-Throughout this tutorial we'll present shell commands together with their output:
-
-- The commands will be prefixed by a
-prompt matching the shell it's launched in. 
-
-  For example, `PS C:\Users\me\tutorial>` is a PowerShell prompt where the current working directory is `C:\Users\me\tutorial`, thus running on Windows.
+WSL enables using a Linux shell and Windows PowerShell side-by-side on the same machine.
+Throughout this tutorial, commands will be prefixed by a prompt that indicates the shell being used, for example:
 
+- `PS C:\Users\me\tutorial>` is a PowerShell prompt where the current working directory is `C:\Users\me\tutorial`.
 
-  Similarly, a Linux shell prompt logged in as root with the current working directory as the root home directory will look like `root@hostname:~#`
+- `root@hostname:~#` indicates a Linux shell prompt with login as root in the root home directory `~`.
 
-- The outputs are preserved to give you more context.
+Output logs are included in this tutorial when instructive but are typically omitted to save space.
 ```
 
-
 ## Set things up
 
-### Install the `Windows Subsystem for Linux` (WSL) application
-
-```{warning}
-**If you already have it pre-installed:** 
-
-Please reset its configuration to the default settings by (making a backup of your existing `~\.wslconfig` file, then) running:
+### Install WSL and Ubuntu
 
+WSL can be installed directly from the [Microsoft Store](https://www.microsoft.com/store/productId/9P9TQF7MRM4R).
```

Comment:
Not exactly a problem with your PR, but I think we need to recommend upstreamm to increase the spacing between the headings, don't you think so?

![image](https://github.com/canonical/ubuntu-pro-for-wsl/assets/11138291/7072d773-c3f4-4e24-bbed-837beab159f4)

Reply:
To my eye I don't have a significant issue with the spacing; however, I agree that any spacing issues should ideally be resolved upstream.
Coming up with our own *ad hoc* solutions adds additional complexity to maintaining the doc and also could result in inconsistent styling across docs from different teams.
In general I would also suggest that we should avoid too many levels of heading. Often this only becomes a requirement when the doc becomes too complex for its own good. It also introduces additional problems with spacing of elements.